certbot/certbot/_internal/main.py

```diff
@@ -503,7 +503,8 @@ def _report_new_cert(config, cert_path, fullchain_path, key_path=None):
                        "expiring certificates. "
                        "We recommend setting up a scheduled task for renewal; see "
                        "https://certbot.eff.org/docs/using.html#automated-renewals "
-                       "for instructions.")
+                       "for instructions."),
+            nl="\n" if config.verb == "run" else "" # visually split output if also deploying
```

Comment:
Sorry. I can't think of a better way.

At least, the output is correct this way. I updated [the before/after doc](https://docs.google.com/document/d/1mwZqpv-VZbeHxV9yQZeI_5Sh9B-iZnApfxB5iEaKvX4/edit#), see the "Success" cases for `certonly` and Apache/nginx.

Reply:
eh, it's not the worst thing we've ever done.